test/cpp/end2end/test_service_impl.h

```diff
@@ -495,8 +522,13 @@ class CallbackTestServiceImpl
     std::unique_lock<std::mutex> lock(mu_);
     return signal_client_;
   }
-  void ClientWaitUntilRpcStarted() { signaller_.ClientWaitUntilRpcStarted(); }
+  int WaitForRpcsWaitingForServerToContinue(
```

Comment:
Same here: Please call this something like `ClientWaitUntilNRpcsStarted()`.

Reply:
done